### DISCUSSION TITLE
Installation and setup help

### DISCUSSION THREAD

**Discussion by AlxSIO** [Security-Onion-Solutions/securityonion#8038] [Unsupported Versions] [answered]
Greetings,
My goal is to test and try Security Onion (so I chose EVAL in the installation) but I'm having some troubles setting it up correctly. I'm using a single Ubuntu 20.04 on a Virtualization Station. I've managed to install SO and connect to the web page, but I don't have anything in Alerts or Hunts and the Disk space keeps filling up, searched for solutions, but none really helped me as I'm a beginner in the domain. I've also searched everywhere for promiscuous mode on Ubuntu 20.04 as it looks like it's needed, but nothing came out of the search and it seems my Virtualization software doesnt offer me to use promiscuous mode (So I can't use  SO ?).
So did I mess up the installation, the configs, or am I missing something ? I've read a lot of the documentation, followed many guides, and yet I'm struggling with the setup and configs.
Thank you.

**Comment by dougburks** [comment, marked as answer]
We'd recommend starting with a simple Import installation following the steps here:
https://docs.securityonion.net/en/2.3/first-time-users.html
